site/en/blog/no-spooky-cookies/index.md

```diff
@@ -0,0 +1,204 @@
+different site then it would be a third-party cookie.
+
+**Remember:** first-party or third-party is all about same-site or cross-site
+context!
+
+The problem with third-party cookies is that they can enable cross-site
+tracking. Instead of setting something like a theme, the shared service might
+store a whole identifier in there. That same identifier is then sent when you
+navigate across different sites that include shared services cookie - which
+means that one service can observe and link your activity across those sites.
+
+{% Img src="image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/vSNgfP4EegQGF1WmKGQa.png", alt="A third-party cookie carrying a unique ID which allows the third-party site to track a user around the web", width="800", height="450" %}
+
+## First-party cookies by default
+
+We've already made progress on our journey here! It used to be that just setting
+a plain cookie: `theme=pumpkins` would be sent in all contexts: same-site or
+cross-site! The majority of sites do only want their cookies to be sent in a
+same-site context. This can be controlled via the `SameSite` attribute on the
+cookie. For example:
+
+```text
+Set-Cookie: theme=bats; SameSite=Lax
+```
+
+This tells the browser to only send the cookie if the resource matches the
+top-level site. However, that meant the site had to specify when it wanted the
+first-party cookie. That's a bit backwards in security terms as really you
+should be asking when you want more privileges - not just getting them by
+default.
+
+So now, `SameSite=Lax` is the default. If you just set `theme=bats` it will only
+be sent in a same-site context.
+
+{% Img src="image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/6HGeQJg31h69UtE9zh4v.png", alt="The default SameSite=Lax value stops a cookie being sent in a third-party context", width="800", height="450" %}
+
+If you want a cross-site or third-party cookie (perhaps you need the theme
+displayed in an embedded widget) then you need to specify:
+
+```text
+Set-Cookie: theme=bats; SameSite=None; Secure
+```
+
+{% Img src="image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/rmrLeyBLQXkkLgRfaN3I.png", alt="The explicit SameSite=None value marks the cookie to be sent in cross-site contexts", width="800", height="450" %}
+
+This tells the browser you want the cookie sent in any cross-site context, but
+we do want to restrict to just secure connections.
+
+## Even tastier first-party cookies
+
+While the default did get a bit better, there's still room for you to improve
+that recipe. Here's a quick taste:
+
+```text
+Set-Cookie:  __Host-theme=bats;
+  Secure;
+  Path=/;
+  HttpOnly;
+  Max-Age=7776000;
+  SameSite=Lax;
+```
+
+That will get you a first-party cookie that stays restricted to just one domain,
+secure connections, no access by JavaScript, automatically expires before it
+goes stale, and (of course!) is only allowed in same-site contexts.
+
+**Read more:** There's plenty of tweaking you can do to suit your taste and
```

Comment:
Make this an aside component of choice please.

Reply:
Will do